StickyTabBarViewController/Classes/StickyViewControllerSupportingTabBarController.swift

```diff
@@ -104,4 +103,16 @@ open class StickyViewControllerSupportingTabBarController: UITabBarController, S
         }
         collapsableVCFlow.expand()
     }
+    
```

Comment:
Please add a pragma mark here, otherwise this falls into the `// MARK: - Public API` category from above.

Reply:
https://github.com/emrepun/StickyTabBarViewController/pull/17/commits/37a93045c6d2eb75e31092b0bf4404ae6f3f8750 @Thurman1776-